arch/arm/boot/dts/tegra30-fujitsu-m532.dts

```diff
@@ -0,0 +1,1462 @@
+/dts-v1/;
+
+/* This dts file describes the Fujitsu Stylistic M532 tablet */
+/* CPU-speedo 5, CPU-process 2, SoC-speedo 2, Core-process 0 */
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/thermal/thermal.h>
+#include "tegra30.dtsi"
+#include "tegra30-cpu-opp.dtsi"
+#include "tegra30-cpu-opp-microvolt.dtsi"
+
+/ {
+	model = "Fujitsu M532 (chagall)";
+	compatible = "fujitsu,m532", "nvidia,tegra30";
+
+	aliases {
+		rtc0 = &pmic;
+		rtc1 = "/rtc@7000e000";
+		display0 = &lcd;
+		display1 = &hdmi;
+		serial0 = &uartd; /* reserved: console */
+		serial1 = &uartc;
+		serial2 = &uartb;
+		mmc0 = &sdmmc4;	/* eMMC */
+		mmc1 = &sdmmc1; /* uSD slot */
+		mmc2 = &sdmmc3; /* WiFi */
+	};
+
+	chosen {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+		
+		bootargs = "console=tty1";
+		stdout-path = "display0";
+
+		framebuffer@abc01000 {
+			compatible = "simple-framebuffer";
+			reg = <0xabc01000 0x3e8a00>;
+			width = <1280>;
+			height = <800>;
+			stride = <2560>;
+			format = "r5g6b5";
+			display = <&lcd>;
+			clocks = <&tegra_car TEGRA30_CLK_DISP1>,
+				 <&tegra_car TEGRA30_CLK_PWM>;
+			panel-supply = <&vdd_pnl1_reg>;
+			backlight-supply = <&vdd_bl_reg>;
+		};
+	};
+
+	memory@80000000 {
+		reg = <0x80000000 0x40000000>;
+	};
+	ram-console {
+		compatible = "android,ram-console";
+		reg = <0xbeb00000 0x100000>;
+	};
+
+	firmware {
+		trusted-foundations {
+			compatible = "tlm,trusted-foundations";
+			tlm,version-major = <2>;
+			tlm,version-minor = <8>;
+		};
+	};
+
+	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+
+		firmware@bfe00000 {
+			// used by TF firmware
+			reg = <0xbfe00000 0x200000>;
+			no-map;
+		};
+
+		ramoops@beb00000 {
+			compatible = "ramoops";
+			reg = <0xbeb00000 0x100000>;
+			record-size = <0x00020000>;
+			console-size = <0x00040000>;
+			ftrace-size = <0x00040000>;
+			ecc-size = <16>;
+		};
+
+		lp0_vec@bddf9000 {
+			reg = <0xbddf9000 0x2000>;	// passed from bootloader (ATAGS/NVIDIA, cmdline)
+		};
+/*
```

Comment:
preferably drop all commented stuff if you didn't intent to uncomment it for chagall

Reply:
The reason I did not drop them yet was because I still have to find out what they do, and if they are part of chagall or not